cibyl/sources/jenkins.py

```diff
@@ -318,62 +316,34 @@ def get_tests(self, **kwargs):
         """
             Get tests for a Jenkins job.
 
-            :returns: container of jobs with the last completed build
-            (if any) and the tests
+            :returns: container of jobs with the selected build(s)
+             and their tests
             :rtype: :class:`AttributeDictValue`
         """
 
-        # Get the jobs with the last completed build (default behavior)
-        jobs_found = self.send_request(
-            self.jobs_last_completed_build_query)["jobs"]
-        jobs_filtered = filter_jobs(jobs_found, **kwargs)
-
-        job_objects = {}
+        if not kwargs.get('builds') and not kwargs.get('last_build'):
```

Comment:
Let's make sure to reuse this code, for example moving it to a method of the Source class in all sources that implement get_tests.

Reply:
> Let's make sure to reuse this code, for example moving it to a method of the Source class in all sources that implement get_tests.

As we talk recently, we will merge these changes, check the Elasticsearch changes and merge it when it's done and then we will move the method to implement in all `get_tests`.

Thanks!